.github/workflows/ci.yml

```
@@ -42,6 +42,8 @@ jobs:
         distribution: temurin
         java-version: "${{ matrix.java }}"
         cache: sbt
+    - if: ${{ matrix.os == 'macos-latest' }}
+      run: brew install sbt
```

I don't know what brew does so I'd rather do something like:

```suggestion
      run: |
        mkdir -p "$HOME/bin/"
        curl -sL https://raw.githubusercontent.com/sbt/sbt/v1.9.9/sbt > "$HOME/bin/sbt"
        export PATH="$PATH:$HOME/bin"
        chmod +x "$HOME/bin/sbt"
```